pack/tdc/tdcp.json

```
@@ -348,6 +348,24 @@
         "type_code": "event",
         "xp": 0
     },
+    {
+        "code": "11050",
+        "cost": 2,
+        "deck_limit": 2,
+        "exceptional": false,
+        "faction_code": "rogue",
+        "illustrator": "Pixoloid Studios",
+        "name": "Luger P08",
+        "pack_code": "tdcp",
+        "position": 50,
+        "quantity": 2,
+        "skill_combat": 1,
+        "slot": "Hand",
+        "text": "Uses (2 ammo).\n[free] Exhaust Luger P08 and spend 1 ammo: <b>Fight.</b> You get +1 [combat] for this attack.\n[action] Spend 1 resource: Replenish all ammo on Luger P08 and ready it.",
```

We should no longer use `[free]`, `[fast]` is the preferred token.

```suggestion
        "text": "Uses (2 ammo).\n[fast] Exhaust Luger P08 and spend 1 ammo: <b>Fight.</b> You get +1 [combat] for this attack.\n[action] Spend 1 resource: Replenish all ammo on Luger P08 and ready it.",
```